web/themes/custom/votegov/src/sass/components/page-content.scss

```diff
@@ -7,12 +8,42 @@
 
 .vote-state-territory-content {
   @include vote-narrow-container;
+  --state-territory-text: inherit;
+  --state-territory-h2: inherit;
+  color: var(--state-territory-text);
+
+  h2 {
+    color: var(--state-territory-h2);
+  }
+
+  [data-theme="contrast"] & {
+    --state-territory-text: #{$base-white};
```

Comment:
see my vid in jira... not sure if this is needed unless im looking at the figma wrong